app/src/main/java/ch/admin/bag/dp3t/home/HomeFragment.java

```diff
@@ -232,24 +206,23 @@ private void setupTracingView() {
 				cardSymptomsFrame.setVisibility(VISIBLE);
 				cardTestFrame.setVisibility(VISIBLE);
 				tracingCard.findViewById(R.id.contacs_chevron).setVisibility(VISIBLE);
-				tracingCard.setOnClickListener(v -> showContactsFragment());
+				tracingCard.setOnClickListener(v ->
```

Comment:
I would not have inlined the `showContactsFragment()` method, as it just blows up the code of `setupTracingView()` and creates another few levels of indentation.

Reply:
I did it because I didn't like that showing the ContactsFragment has its own method, while just below showing the ReportsFragment is inlined. So I'll "outline" both.